### DISCUSSION TITLE
Combine/extend hover

### DISCUSSION THREAD

**Discussion by Emiliano-Bucci** [chakra-ui/chakra-ui#5744] [Q&A]
Hi! Is there a way to combine multiple _hover declarations without overriding it? So for example, i have this scenario:

```
const overrides = {
   components: {
       Button: {
          variants: {
              primary: {
                _hover: { // custom styles... }
              }
          }
       }
   }
}

export default extendTheme(overrides);
```

Then when i use the component:

```
<Button variant="primary" _hover={{// extend variant styles...}}>My Button</Button>
```

If i declare `_hover` the original styles are removed and only applied the one i pass in the component. I've also tried to use a function in `Button.variants.primary` and pass the custom _hover declaration without success. Is there a way to do it? Thanks

**Comment by DanieleMeeters** [comment]
+1

I have the same issue 😔
